test-a11y/index.js

```diff
@@ -17,7 +17,7 @@ let chromeOptions = {};
 if (process.env.CI) {
   chromeOptions = { args: ['--headless'] };
 } else {
-  chromeOptions = { args: ['--start-maximized', '--incognito'] };
+  chromeOptions = { args: ['--incognito', '--window-size=768,1024'] };
```

Comment:
Could you explain, does this mean that it looks at the window sizes 768 and 1024?

I'm wondering if its possible to check the screen at all breakpoint sizes that we have, because changes can happen at all of them:

// Grid breakpoints
$pf-global--breakpoint--xs: 0 !default;
$pf-global--breakpoint--sm: 576px !default;
$pf-global--breakpoint--md: 768px !default;
$pf-global--breakpoint--lg: 992px !default;
$pf-global--breakpoint--xl: 1200px !default;
$pf-global--breakpoint--2xl: 1450px !default;

Reply:
@christiemolloy that will set the viewport to 768px (w) by 1024px (h) . To extend the audit to check for each of the supported breakpoints is a bit beyond the scope of this PR as we'd then want to make sure we have a way of collecting and publishing the results for each of the breakpoints. Currently, we're just using a single reporting mechanism and I found that cranking down the viewport size in general does wonders for bringing alignment between local dev and CI.

We have these two issues in the backlog which would be a good point in time to address checking all supported breakpoints: https://github.com/patternfly/patternfly-next/issues/635 and https://github.com/patternfly/patternfly-next/issues/649

Reply:
Ok thanks for clarifying that! Sounds good to me :)